Runtime/painting/text_painter.cs

```diff
@@ -43,9 +42,9 @@ public class TextPainter {
                 }
 
                 this._textScaleFactor = value;
-                this._paragraph = null;
                 this._layoutTemplate = null;
                 this._needsLayout = true;
+                Paragraph.release(ref this._paragraph);
```

Comment:
According to my thinking, although we release this._paragraph manually here and there, but when text_painter is disposed this._paragraph will not be release?

Any checks done on this stuff yet? Please make sure there is not any memory leak in the Paragraph pool.

Reply:
No, paragraph attached to TextPainter will have to be handled directly to GC due to its indefinite lifecycle. Releasing it in ~TextPainter() will cause it to be added back to pool after GC, but before that new paragraphs would have to be created.

Reply:
I think there will not be memory leak, unless from some time paragraphs not used any more, leaving the existing paragraphs in the pool useless.